bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/ProductImpl.java

```diff
@@ -464,6 +465,11 @@ public String getDataLayerDescription() {
 
     @Override
     public CategoryData[] getDataLayerCategories() {
+        List<CategoryInterface> productCategories = productRetriever.fetchProduct().getCategories();
+
+        if (productCategories == null || productCategories.size() == 0)
```

Comment:
I know it's picky, but we use brackets `{}` even for 1-liner blocks.

Reply:
Fixed